Composer/packages/client/src/recoilModel/dispatchers/project.ts

```diff
@@ -251,6 +251,7 @@ export const projectDispatcher = () => {
           callback(projectId);
         }
       } catch (ex) {
+        console.log(ex);
         set(botProjectIdsState, []);
         removeRecentProject(callbackHelpers, path);
         handleProjectFailure(callbackHelpers, ex);
```

Comment:
If we need to log errors to the console, I would hope that methods handleProjectFailure do it rather than each catch.

Reply:
It already does that. The console.error can be safely removed

Reply:
This is used to test, I forget to remove it. I should be removed.